[2024-06-25T08:38:52.094+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-25T08:38:52.200+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_snowflake_v4.dbt_run scheduled__2024-06-24T00:00:00+00:00 [queued]>
[2024-06-25T08:38:52.221+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_snowflake_v4.dbt_run scheduled__2024-06-24T00:00:00+00:00 [queued]>
[2024-06-25T08:38:52.221+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-25T08:38:52.273+0000] {taskinstance.py:2330} INFO - Executing <Task(DockerOperator): dbt_run> on 2024-06-24 00:00:00+00:00
[2024-06-25T08:38:52.299+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dbt_snowflake_v4', 'dbt_run', 'scheduled__2024-06-24T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpj2d7ayh1']
[2024-06-25T08:38:52.307+0000] {standard_task_runner.py:91} INFO - Job 6: Subtask dbt_run
[2024-06-25T08:38:52.319+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=940) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-25T08:38:52.321+0000] {standard_task_runner.py:63} INFO - Started process 943 to run task
[2024-06-25T08:38:52.430+0000] {task_command.py:426} INFO - Running <TaskInstance: dbt_snowflake_v4.dbt_run scheduled__2024-06-24T00:00:00+00:00 [running]> on host 40877dbc4b09
[2024-06-25T08:38:52.694+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dbt_snowflake_v4' AIRFLOW_CTX_TASK_ID='dbt_run' AIRFLOW_CTX_EXECUTION_DATE='2024-06-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-24T00:00:00+00:00'
[2024-06-25T08:38:52.695+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-25T08:38:52.806+0000] {docker.py:366} INFO - Starting docker container from image ghcr.io/dbt-labs/dbt-snowflake:latest
[2024-06-25T08:38:52.829+0000] {docker.py:374} WARNING - Using remote engine or docker-in-docker and mounting temporary volume from host is not supported. Falling back to `mount_tmp_dir=False` mode. You can set `mount_tmp_dir` parameter to False to disable mounting and remove the warning
[2024-06-25T08:39:01.424+0000] {docker.py:436} INFO - [0m08:39:01  Running with dbt=1.8.2
[2024-06-25T08:39:02.552+0000] {docker.py:436} INFO - [0m08:39:02  Registered adapter: snowflake=1.8.3
[2024-06-25T08:39:03.629+0000] {docker.py:436} INFO - [0m08:39:03  Found 5 models, 9 data tests, 2 sources, 561 macros
[2024-06-25T08:39:03.633+0000] {docker.py:436} INFO - [0m08:39:03
[2024-06-25T08:39:05.657+0000] {docker.py:436} INFO - [0m08:39:05  Concurrency: 1 threads (target='dev')
[2024-06-25T08:39:05.659+0000] {docker.py:436} INFO - [0m08:39:05
[2024-06-25T08:39:05.675+0000] {docker.py:436} INFO - [0m08:39:05  1 of 5 START sql view model dbt_schema.stg_tpch_line_items ..................... [RUN]
[2024-06-25T08:39:06.778+0000] {docker.py:436} INFO - [0m08:39:06  1 of 5 OK created sql view model dbt_schema.stg_tpch_line_items ................ [[32mSUCCESS 1[0m in 1.10s]
[2024-06-25T08:39:06.781+0000] {docker.py:436} INFO - [0m08:39:06  2 of 5 START sql view model dbt_schema.stg_tpch_orders ......................... [RUN]
[2024-06-25T08:39:07.622+0000] {docker.py:436} INFO - [0m08:39:07  2 of 5 OK created sql view model dbt_schema.stg_tpch_orders .................... [[32mSUCCESS 1[0m in 0.84s]
[2024-06-25T08:39:07.626+0000] {docker.py:436} INFO - [0m08:39:07  3 of 5 START sql table model dbt_schema.int_order_items ........................ [RUN]
[2024-06-25T08:39:22.266+0000] {docker.py:436} INFO - [0m08:39:22  3 of 5 OK created sql table model dbt_schema.int_order_items ................... [[32mSUCCESS 1[0m in 14.64s]
[2024-06-25T08:39:22.268+0000] {docker.py:436} INFO - [0m08:39:22  4 of 5 START sql table model dbt_schema.int_order_items_summary ................ [RUN]
[2024-06-25T08:39:23.992+0000] {docker.py:436} INFO - [0m08:39:23  4 of 5 OK created sql table model dbt_schema.int_order_items_summary ........... [[32mSUCCESS 1[0m in 1.72s]
[2024-06-25T08:39:23.998+0000] {docker.py:436} INFO - [0m08:39:23  5 of 5 START sql table model dbt_schema.fct_orders ............................. [RUN]
[2024-06-25T08:39:26.020+0000] {docker.py:436} INFO - [0m08:39:26  5 of 5 OK created sql table model dbt_schema.fct_orders ........................ [[32mSUCCESS 1[0m in 2.02s]
[2024-06-25T08:39:26.026+0000] {docker.py:436} INFO - [0m08:39:26
[2024-06-25T08:39:26.028+0000] {docker.py:436} INFO - [0m08:39:26  Finished running 2 view models, 3 table models in 0 hours 0 minutes and 22.39 seconds (22.39s).
[2024-06-25T08:39:26.131+0000] {docker.py:436} INFO - [0m08:39:26  
[0m08:39:26  [32mCompleted successfully[0m
[0m08:39:26  
[0m08:39:26  Done. PASS=5 WARN=0 ERROR=0 SKIP=0 TOTAL=5
[2024-06-25T08:39:29.590+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-25T08:39:29.673+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dbt_snowflake_v4, task_id=dbt_run, run_id=scheduled__2024-06-24T00:00:00+00:00, execution_date=20240624T000000, start_date=20240625T083852, end_date=20240625T083929
[2024-06-25T08:39:29.737+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-25T08:39:29.789+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-25T08:39:29.792+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
